### DISCUSSION TITLE
error : static  assertion failed with "MIN_GPU_ARCH=610 must bound __CUDA_ARCH__=520 from below, but doesn't."

### DISCUSSION THREAD

**Discussion by marcilein98** [NVlabs/instant-ngp#248] [Q&A]
When building "cmake . -B build" is successful but with "cmake --build build --config RelWithDebInfo -j 16" I get the error: static  assertion failed with "MIN_GPU_ARCH=610 must bound __CUDA_ARCH__=520 from below, but doesn't."
I have installed:
Python 3.9
Cuda 11.6
Visual Studio 2019 16.11
Optix 7.4.0
My Graphics Card is GTX 1050 TI

Here's the Log: https://pastebin.com/kczC1SU4

**Comment by timmeh4242** [comment]
Had the same error - fetched the latest changes (there were several today), updated submodules, deleted the build directory, then re-ran both build commands. I've also set the CUDA_VISIBLE_DEVICES and TCNN_CUDA_ARCHITECTURES to the corresponding values listed [here](https://github.com/NVlabs/instant-ngp#compilation-windows--linux).

It now builds successfully, however I get the following when trying to run the test scenes:

"ERROR    Uncaught exception: Could not allocate memory: \instant-ngp\dependencies\tiny-cuda-nn\include\tiny-cuda-nn/gpu_memory.h:109 cudaMalloc(&rawptr, n_bytes+DEBUG_GUARD_SIZE*2) failed with error no CUDA-capable device is detected"

UPDATE: I've re-installed the CUDA toolkit, removed the environment variables I'd previously added to get things building, restarted the machine, and all seems good now 👍
